helpers/compile/build.ts

```diff
@@ -4,14 +4,15 @@ import { flatten } from '../blaze/flatten'
 import { pipe } from '../blaze/pipe'
 import { map } from '../blaze/map'
 import { transduce } from '../blaze/transduce'
-import glob from 'glob'
+import glob from 'globby'
```

Comment:
Was this import broken before?

Reply:
Although they are compatible, `globby` had a gitignore option. Thinking about it, it is not even needed anymore, so I could revert that in a later PR.